examples/editor/index-new.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CARTO-VL EDITOR</title>
+
+    <!-- Remove jquery -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+    <script src='https://libs.cartocdn.com/mapbox-gl/v0.45.0-carto1/mapbox-gl.js'></script>
+    <link href='https://libs.cartocdn.com/mapbox-gl/v0.45.0-carto1/mapbox-gl.css' rel='stylesheet' />
+    <link rel="stylesheet" href="./css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
+    <script src="../../dist/carto-vl.js"></script>
```

Comment:
Link to CDN, pls

Reply:
IMO the editor should point to the most recent compiled version, this is nice for testing. On the other side, I agree with you the generated HTML should point to a fixed CDN version.

Reply:
That has the potential problem of generating a different result but... I don't know, it makes sense. In any case, the generated html needs to point to a fixed version.

Reply:
You sure? https://github.com/CartoDB/carto-vl/pull/567/files/7fda9881be818fec82c7253c6e68de62e72fe538#diff-c490479a620b105cad06cf37923e6118R424 :slightly_smiling_face:

Reply:
https://github.com/CartoDB/carto-vl/pull/567/commits/b9656ed322400f0adfd3d5bbc74c94bb23b2771f

Reply:
I just changed this this morning :)